tests/util/run_graph_engine.cpp

```diff
@@ -42,10 +42,10 @@ std::vector<core::conversion::InputRange> toInputRangesDynamic(std::vector<at::T
 }
 
 std::vector<at::Tensor> RunEngine(std::string& eng, std::vector<at::Tensor> inputs) {
-  LOG_DEBUG("Running TRT version");
-  auto engine_ptr = c10::make_intrusive<trtorch::core::runtime::TRTEngine>("test_engine", eng);
-  auto outputs = trtorch::core::runtime::execute_engine(inputs, engine_ptr);
-  return outputs;
+    LOG_DEBUG("Running TRT version");
```

Comment:
We probably need to run this PR through the linter

Reply:
Ran a linter